### ISSUE TITLE
vkontakte: author information for posts

### ISSUE THREAD

**Issue by hgrsd** [JustAnotherArchivist/snscrape#503]
Hi there!

I'm wondering whether you're open to adding author information (id) to posts (both originals and quoted posts). The information appears to be available in the scraped data.

E.g. (for non-quoted):
```
<a class="author" data-from-id="123456" href="/someuser">
  Some User
</a>
```

or for a quoted post:

```
<a class="copy_author" data-parent-post-id="123455_1" data-post-id="222222_111" href="/some href">
  Some Quoted User
</a>
```

Could I open a PR to add this information?

**Comment by JustAnotherArchivist** [comment]
Yes, that would be good to add. The `VKontaktePost` object would have a `user` or `author` attribute of type `User`.
Some care is needed though. It appears that for non-quote posts, the link doesn't indicate whether the user is verified or not, but for quotes it does. I think the best option would be to make the `verified` field on `User` optional and then leave it as `None` for the non-quote posts. For quote posts, it should always be a bool though.
